src/stylesheets/app.scss

```diff
@@ -1,115 +1,155 @@
 @import "base/typography";
 @import url('https://fonts.googleapis.com/css?family=Montserrat:300,300i,400,400i');
-* {
-    box-sizing: border-box;
+
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
 }
 
 body {
-    background-color: #DEEDF0;
+  background-color: #DEEDF0;
```

Comment:
Now that you have things moved over to sass, how about you start to move some of these colors into variables